diff -urN md4sum-0.02.03/Makefile.Linux md4sum-0.02.03.new/Makefile.Linux
--- md4sum-0.02.03/Makefile.Linux	2007-04-11 23:15:46.000000000 +0200
+++ md4sum-0.02.03.new/Makefile.Linux	2007-05-13 00:42:45.000000000 +0200
@@ -3,8 +3,8 @@
 #
 
 CC=gcc
-CFLAGS=-Wall -Wstrict-prototypes -O2 -fomit-frame-pointer -pipe -I$(HDDIR) -DOS_TYPE_Linux -DOS_TYPE_STR=\"Linux\" -Dprg_name=\"$(PRG)\" -DMANUAL_NUMBER=$(MAN_NO)
-LDFLAGS=-Wl,-warn-common -s -L$(LIBDIR)
+override CFLAGS+=-Wall -Wstrict-prototypes -O2 -fomit-frame-pointer -pipe -I$(HDDIR) -DOS_TYPE_Linux -DOS_TYPE_STR=\"Linux\" -Dprg_name=\"$(PRG)\" -DMANUAL_NUMBER=$(MAN_NO)
+override LDFLAGS+=-Wl,-warn-common -L$(LIBDIR)
 
 LDLIBS = 
 HDDIR = ./
